packages/dom/src/tabbable.js

```diff
@@ -133,3 +133,47 @@ export function find( context ) {
 		.map( mapObjectTabbableToElement )
 		.reduce( createStatefulCollapseRadioGroup(), [] );
 }
+
+/**
+ * Given an element, find the preceding tabbable elemnt.
+ *
+ * @param {Element} element The element before which to look.
+ */
+export function findPrevious( element ) {
+	const focusables = findFocusable( document.body );
+	const index = focusables.indexOf( element );
```

Comment:
Should we assume that `element` is focusable? What happens if it's not? What if I'm interested in finding the previous/next focusable relative to a non-focusable element? Or would we consider that unsupported? Can we make this clear in the function documentation?

I might also wonder about the role of "context", where in the existing `find` we assume a context must be passed. Here we force that it can't be passed and assume a default of `document.body`. Should we allow for a default in `find`? Should we allow for an explicit `context` to this function, in order to allow the candidates to be constrained? (It could be passed directly into `findFocusable`).

Reply:
Yes, I would assume a focusable, because normally you find a previous and next focusable _from_ the current focusable. I think that's a fine assumption to make? I'll rewrite this a bit so we rest `find`.

Reply:
I added a comment.